src/routes/__layout.tsx

```diff
@@ -34,11 +36,13 @@ export default function RootRoute({ children }: TuonoLayoutProps): JSX.Element {
         <link rel="manifest" href="/site.webmanifest" />
       </head>
       <body>
-        <PostHogProvider>
-          <PostHogPageView />
-          <App>{children}</App>
-        </PostHogProvider>
-        <TuonoScripts />
+        <ThemeProvider>
+          <PostHogProvider>
```

Comment:
Posthog should go above the theme provider

```suggestion
        <PostHogProvider>
          <ThemeProvider>
```